client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfoInterfaces.ts

```diff
@@ -1,3 +1,6 @@
+import { Dispatch, SetStateAction } from "react";
+
 export interface InvestigatedPersonInfoOutcome {
-    confirmExitUnfinishedInvestigation: () => void;
+    confirmExitUnfinishedInvestigation: (epidemiologyNumber: string, cantReachInvestigated: boolean) => void;
```

Comment:
why do you need cantReachInvestigated to finish investigation?

Reply:
Because if I cant reach investigated, the status changes

Reply:
for that you've got handleCantReachInvestigatedCheck no?